src/table/components/head/MenuList/MenuGroup.tsx

```diff
@@ -43,13 +99,15 @@ const MenuGroupItems = ({ id, onClick, itemTitle, icon, enabled, subMenus }: Hea
   };
 
   return (
-    <div key={id}>
+    <>
```

Comment:
when git rid of div, MUI can work with keyboard navigation well using correct props except ArrowDown and ArrowUp